### PR TITLE
Fix noninteger sizing/indexing (#776)

### PR DESCRIPTION
**Pull request by titodalcanton**

### CODE REVIEW

**Comment by ahnitz** [comment]
Tito, are you still working on this? Or would you like someone to look over it and approve it? Nobody is assigned at the moment.

**Comment by titodalcanton** [comment]
I'm doing an extensive check of the change because `next_power_of_2()` does not return the same as before and also I need to try this on newer numpy than I have on atlas. I'll assign you or Ian when I'm done.

**Comment by titodalcanton** [comment]
Hi Alex, I verified that the gates are exactly as before on all O1 data, and that the patch fixes the warning with the latest numpy. Please feel free to merge.

File: pycbc/strain.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
 from scipy.signal import kaiserord
  
 
-next_power_of_2 = lambda n: 2 ** (math.ceil(math.log(n, 2)) + 1)
+next_power_of_2 = lambda n: 1<<(n).bit_length()
```
Comment (spxiwh) [inline pycbc/strain.py]: What's the reason for this change? To me this doesn't seem clearer, and speed is not an issue with this function (probably this should be a function, not a lambda).

Comment (titodalcanton) [inline pycbc/strain.py]: That guarantees that the result is an int, and also gives more correct results (try plotting both versions as a function of n). To me it also looks clearer, but that might be personal taste :)
